flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/systems.cfg

```diff
@@ -73,32 +73,19 @@ lightdef.19 = Type:10#Index:2#LocalPosition:34.8,1.0,3.7#LocalRotation:270,500,0
 lightdef.20 = Type:11#Index:1#LocalPosition:35.2,0,8.5#LocalRotation:80,0,0#EffectFile:LIGHT_A32NX_Pedestal#PotentiometerIndex:76
 
-; Captain left glareshield light
-lightdef.21 = Type:12#Index:3#LocalPosition:38.97,-2.550,5.87#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
-
-; Captain right glareshield light
-lightdef.22 = Type:12#Index:3#LocalPosition:38.97,-0.6,6.2#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
-
-; F/O left glareshield light
-lightdef.23 = Type:12#Index:4#LocalPosition:38.97,0.7,6.2#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
-
-; F/O right glareshield light
-lightdef.24 = Type:12#Index:4#LocalPosition:38.97,2.55,5.87#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
-
-; Overall glareshield (fake) ambient light
-lightdef.25 = Type:12#Index:3#LocalPosition:36.5,0,6.5#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiant#PotentiometerIndex:83
-
-; Captain left glareshield (fake) ambient light
-lightdef.26 = Type:12#Index:3#LocalPosition:37.97,-2.45,4.87#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiantLow#PotentiometerIndex:83
-
-; Captain right glareshield (fake) ambient light
-lightdef.27 = Type:12#Index:3#LocalPosition:38,-0.6,4.8#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiantLow#PotentiometerIndex:83
-
-; F/O left glareshield (fake) ambient light
-lightdef.28 = Type:12#Index:4#LocalPosition:38,0.7,4.8#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiantLow#PotentiometerIndex:83
-
-; F/O right glareshield (fake) ambient light
-lightdef.29 = Type:12#Index:4#LocalPosition:37.97,2.45,4.87#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiantLow#PotentiometerIndex:83
+; Main panel flood lights
+lightdef.21 = Type:12#Index:3#LocalPosition:38.97,-2.6,5.87#LocalRotation:-10,0,-8#EffectFile:LIGHT_A32NX_MainPanelFloodEnd#PotentiometerIndex:83   ; CPT L
+lightdef.22 = Type:12#Index:3#LocalPosition:38.97,-0.6,6.2#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_MainPanelFloodCenter#PotentiometerIndex:83  ; CPT R
+lightdef.23 = Type:12#Index:4#LocalPosition:38.97,0.7,6.2#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_MainPanelFloodCenter#PotentiometerIndex:83   ; F/O L
+lightdef.24 = Type:12#Index:4#LocalPosition:38.97,2.6,5.87#LocalRotation:-10,0,8#EffectFile:LIGHT_A32NX_MainPanelFloodEnd#PotentiometerIndex:83     ; F/O R
+
+; Main panel flood lights ambient fx
+lightdef.25 = Type:12#Index:3#LocalPosition:38.8,-2.58,5.78#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_MainPanelFloodAmbientEnd#PotentiometerIndex:83   ; CPT
+lightdef.26 = Type:12#Index:3#LocalPosition:38.75,0,5.8#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_MainPanelFloodAmbientCenter#PotentiometerIndex:83    ; Center
+lightdef.27 = Type:12#Index:4#LocalPosition:38.8,2.58,5.78#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_MainPanelFloodAmbientEnd#PotentiometerIndex:83    ; F/O
```

Comment:
How does this play out when certain buses are unpowered? At the moment it seems a situation in which the CPT side is unpowered will also lead to all the ambient light in the centre disappearing, while that maybe shouldn't be the case. Similarly, when the F/O side is out, the centre might have too much light.

Reply:
Hi @davidwalschots, I do not understand, all these lights are on the same type (`Type:12`)? Isn't that the bus?
I know you have been into this file and done work here earlier. Enlighten me please and/or suggest a change.

` ; light.n = CircuitType, ZOffset, XOffset, YOffset, XRotationInDegrees, YRotationInDegrees, ZRotationInDegrees, fxfile`

Reply:
No, Type:12 is not equal to bus 12. From the documentation I wrote within the `systems.cfg` file:

>; The lightdef.Type integer and circuit.Type enum are related. Below we list the
; known relationships between these values.
;
; lightdef.Type = circuit.Type
; 1 = CIRCUIT_LIGHT_BEACON
; 2 = CIRCUIT_LIGHT_STROBE
; 3 = CIRCUIT_LIGHT_NAV
; 4 = CIRCUIT_LIGHT_PANEL
; 5 = CIRCUIT_LIGHT_LANDING
; 6 = CIRCUIT_LIGHT_TAXI
; 7 = CIRCUIT_LIGHT_RECOGNITION
; 8 = CIRCUIT_LIGHT_WING
; 9 = CIRCUIT_LIGHT_LOGO
; 10 = CIRCUIT_LIGHT_CABIN,
; 11 = CIRCUIT_LIGHT_PEDESTAL
; 12 = CIRCUIT_LIGHT_GLARESHIELD
;
; You will find other types being used, but it is unclear how they relate. For example,
; lightdef.Type 13 seems to be always on, and can only be brightened by adapting the associated potentiometer.
;
; lightdef.Index and circuit.Type are related. When the lightdef.Index and circuit.Type numeric suffix match,
; then that circuit affects the powered state of the light. For example, these lightdef and circuit definitions
; are related:
; lightdef.35 = Type:4#Index:5
; circuit.81 = Type:CIRCUIT_LIGHT_PANEL:5#Connections:bus.7
;
; When electrical bus 7 is powered, then circuit 81 is powered, which in turn can power lightdef 35.

Thus, all lights with `Type:12#Index:3` are powered by `circuit.60 = Type:CIRCUIT_LIGHT_GLARESHIELD:3`; and all lights with `Type:12#Index:4` are powered by `circuit.82 = Type:CIRCUIT_LIGHT_GLARESHIELD:4`.

Thus, when DC1 is unpowered, `lightdef.23`, `lightdef.24`, and `lightdef.27` won't illuminate. When DC ESS is unpowered, `lightdef.21`, `lightdef.22`, `lightdef.25`, and `lightdef.26` won't illuminate.

You can test situations in which theses buses (but not the other) become unpowered by:

**DC ESS without DC1**
Fly and turn off both ENG GENs (while APU GEN is not available either). AC1, AC2 and therefore DC1 and DC2 will be unavailable. DC ESS is powered initially by the batteries and later by the EMER GEN.

**DC1 without DC ESS**
- Push the BUS TIE to OFF.
- Push AC ESS FEED to ALTN.
- Push GEN2 to OFF.

Results in AC1, DC1, DC BAT and DC2 being powered, while AC 2, AC ESS, and DC ESS become unpowered.

Reply:
@davidwalschots Thanks for valuable insight and explanation. Will seek ground truth, and make sure they are powered correctly.